app/src/main/java/io/github/droidkaigi/confsched2017/view/helper/DataBindingHelper.java

```diff
@@ -78,6 +78,14 @@ private static void setImageUrlWithSize(ImageView imageView, @Nullable String im
         }
     }
 
+    private static String convertToFullImageUrl(String imageUrl) {
+        if (imageUrl.startsWith("/")) {
+            return "https://droidkaigi.github.io/2017" + imageUrl;
```

Comment:
Ah, sorry, I think this should be converted in ViewModel.
Can you refactor on another PR? 🙇